### PR TITLE
Switch tests to one step coldstart

### PR DESCRIPTION
**Pull request by DeniseWorthen**
### Description of changes
Implement 1-step cold start for all tests

### Specific notes

Issues Fixed (include github issue #):
#185 
#186 
#179 

Are changes expected to change answers?
- New baselines are required for all tests (except for previous 1-step coldstart test)

Specific changes:
- changes in parm directory (yes)
- changes in module files (no)
- new tests added or removed (yes)

All "cold" tests removed. 
Default for 'use_coldstart' is set false.
Name of IC for CICE is change from 'cice5' to 'cice' except for the 35d tests (all 168 files are named 'cice5')

Testing performed:
- machines: hera,orion,dell

- The existing 1-step coldstart test reproduced.

- 2 of the 8 35d tests for waves and no waves were run for 5d each (a total of 4 tests) and all successfully started and finished the 5d runs. This was a test to ensure that changes to rt.sh did not break the 35d tests.

Hashes used for testing:
- NEMS:9d0517
- CMEPS:5e63bd
- FV3:6bc61d
- MOM6:3ce81f
- CICE:285985
- WW3:96e3f3a
- FMS:f61416
- stochastic_physics:ffdd19

Co-authored by:

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
New baselines will be generated on hera,orion and dell.

File: tests/fv3_conf/fv3_slurm.IN [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```
Comment (junwang-noaa) [inline tests/fv3_conf/fv3_slurm.IN]: This is a temporary file created from fv3_slurm.IN_platform, I suggest not to commit it.

Comment (DeniseWorthen) [reply tests/fv3_conf/fv3_slurm.IN]: I was wondering why it was there. I will remove.

**Comment by DeniseWorthen** [comment]
Ready for review.

**Review by binli2337 — APPROVED** [review]
Looks good.

**Review by junwang-noaa — APPROVED** [review]
The changes look good. I think parm/nems.configure\*cold\* can also be deleted, also cpld_fv3_ccpp_mom6_cice_cmeps_1stepcold_atm_flux and cpld_fv3_ccpp_mom6_cice_cmeps_2d_atm_flux are the same now, cpld_fv3_ccpp_mom6_cice_cmeps_1stepcold_atm_flux can be deleted, we can do these in next commit.

**Pull request merged**